ci-operator/step-registry/ignition/bastionhost/ignition-bastionhost-commands.sh

```diff
@@ -457,3 +462,6 @@ cp "${bastion_ignition_file}" "${SHARED_DIR}/${CLUSTER_NAME}-bastion.ign"
 echo "Ignition file '${SHARED_DIR}/${CLUSTER_NAME}-bastion.ign' created"
 
 rm -rf "${workdir}"
+git clone https://github.com/magnific0/wondershaper.git
+cd wondershaper
+./wondershaper -a eth0 -u 4096 -d 8192
```

Comment:
Same as https://github.com/openshift/release/pull/52409/files#r1642074253, and please use the absolute path instead.

Reply:
then we need to move the code before "rm-rf  ${workdir}" .

Reply:
@yunjiang29, what you mean is using ${workdir}/wondershaper/wondershaper? this is a workaround way to limited the bandwitdth of nic eth0, we don't plan to use this formally, the purpose that we submit this pr is creating a bastion host and check if limited bandwidth affect rosa cluster creation. once we get result, we will close this PR.

Reply:
OK